sycl/include/sycl/sycl.hpp

```diff
@@ -96,4 +96,8 @@
 // We used to include those and some code might be reliant on that.
 #include <cmath>
 #include <complex>
+#include <istream>
```

Comment:
Are that intended includes? Probably done for test debugging but then it is not clear that test passes on ci.
Test description:
// Test that only iosfwd and not istream, ostream, and iostream_proxy
// are included.
#include <sycl/sycl.hpp>

Reply:
The test is with `SYCL2020_CONFORMANT_APIS` defined. This section will be removed during the next API breaking window.